### PR TITLE
new pull request to update 'programme/index.html' 

### PR DESCRIPTION
**Pull request by SimonNtz**
The former had 11 commit -due to issues after rebase

### CODE REVIEW

**Comment by rspt** [comment]
Sorry but I really don't like when you merge wrong code using PR ;)

File: programme/index.html [hunk 1]
```diff
@@ -93,6 +93,8 @@
 
   <h4><i>Les personnes à mobilité réduite</i></h4>
   <p class="text-justify"></p>
+    Malgré l'effort entrepris depuis quelques années, le retard pris dans la réalisation de certains équipements est malheureusement patent: ascenseurs inadaptés, bordures de trottoirs insuffisamment abaissées, absence de seuils adaptés pendant des travaux, caisses placées trop en hauteur dans les parkings, notable insuffisance de places réser-vées aux handicapés, panneaux de stationnement non visibles, véhicules stationnant sur les trottoirs, chemins impraticables dans les parcs (racines). On le voit, cette liste non exhaustive est longue. Un sérieux effort dans donc être entrepris afin de permet-tre aux handicapés de pouvoir se déplacer partout dans nos communes.
```
Comment (rspt) [inline programme/index.html]: Just look at the line 95. You close the `p` element before... So it's not a valid HTML page anymore...

**Comment by rspt** [comment]
@SimonNtz t'oublies pas de corriger ça stp ;)

File: programme/index.html (hunk 1 above)
Comment (rspt) [inline programme/index.html]: And you also have a typo: "réservées" and not "réser-vées"

Comment (SimonNtz) [reply programme/index.html]: Thanks for the feedback, the correction is done in a new PR.

**Pull request merged**